programmers/mock-test.js

```diff
@@ -0,0 +1,30 @@
+function solution(answers) {
+    let collect1 = 0;
+    let collect2 = 0;
+    let collect3 = 0;
```

Comment:
정답을 담을 배열,
수포자1, 수포자2, 수포자3이 찍는 규칙을 찾아서 배열로 선언해줍니다.
수포자1, 수포자2, 수포자3의 카운트를 세는 배열을 선언해줍니다.
가장 문제를 많이 맞힌 번호를 담을 max 변수를 선언해줍니다.
```suggestion
    const answer = [];
    const personOne = [1, 2, 3, 4, 5];
    const personTwo = [2, 1, 2, 3, 2, 4, 2, 5];
    const personThree = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
    const count = [0, 0, 0];
    let max = 0;
    
```